ecl/hql/hqlmanifest.cpp

```diff
@@ -198,6 +198,31 @@ void ResourceManifest::loadResource(const char *filepath, MemoryBuffer &content)
     read(fio, 0, (size32_t) f->size(), content);
 }
 
+void queryJfrog(const char *localpath, IPropertyTree &item)
+{
+    const char *repo = item.queryProp("@repository");
+    if(repo)
+    {
+        StringBuffer filename(localpath);
```

Comment:
question: Should localpath be passed in, or should this function be responsible for creating a temporary local path in a valid location?  (I don't know what typical requirements would be.)

Reply:
For now, I have created a temporary directory under /tmp/HPCCSystems/jfrog_downloads/

Reply:
I think this should be using 

    getTempFilePath(tmpDirName, "jfrog", nullptr);

or similar to create the temporary directory.